libvast/src/system/count_command.cpp

```diff
@@ -41,7 +41,7 @@ namespace vast::system {
 
 caf::message
 count_command(const command::invocation& invocation, caf::actor_system& sys) {
-  VAST_TRACE(invocation);
+  VAST_DEBUG_ANON(invocation);
```

Comment:
```suggestion
  VAST_TRACE(invocation);
```

Looks like a leftover debug change.

Reply:
This is actually in a separate commit - I think it's low-frequency and high-value debug information, so I've been promoting these where i've noticed them.

Reply:
Fair enough.